runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -615,6 +617,11 @@ public static StreamingDataflowWorker fromDataflowWorkerHarnessOptions(
     this.timeAtMaxActiveThreads =
         pendingCumulativeCounters.longSum(
             StreamingSystemCounterNames.TIME_AT_MAX_ACTIVE_THREADS.counterName());
+    this.activeThreads =
+        pendingCumulativeCounters.intSum(StreamingSystemCounterNames.ACTIVE_THREADS.counterName());
+    this.maxActiveThreads =
+        pendingCumulativeCounters.intSum(
```

Comment:
intMax?

Reply:
+1

Reply:
We want just max number threads (more in next comment)

Reply:
I don't think it makes sense to sum/aggregate thread # in general. This will monotonically increase for each call